tests/system/node/0/config.yaml

```diff
@@ -53,8 +53,8 @@ validator:
 ################################################################################
 network:
   # Supported value: IPv4, IPv6
-  - https://node-2:2735
-  - https://node-3:3735
-  - https://node-4:4735
-  - https://node-5:5735
-  - https://node-6:6735
+  - http://node-2:2735
+  - http://node-3:3735
+  - http://node-4:4735
+  - http://node-5:5735
+  - http://node-6:6735
```

Comment:
Why??

Reply:
There is an explanation in the commit message.

Reply:
So were the nodes not using TLS, even with the `https`?